README.md

```diff
@@ -1,8 +1,33 @@
 # AzureBot
-This is the home of the code used to run the AzureBot and is built with the [Microsoft Bot framework] (http://docs.botframework.com/), [Microsoft Bot Builder C# SDK](http://docs.botframework.com/sdkreference/csharp/), and the [Azure Resource Manager Nuget package](https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager).
+The AzureBot was created to improve the productivity of any developer, admin, or team working with Azure. It is not currently publicly available, but you can follow the [Developer Set Up](#developersetup) to run it yourself and contribute.
```

Comment:
developersetup link doesn't work.  I'd either remove the link as-is or just link to the actual developer setup guide as you did below.